db.tf

```diff
@@ -45,22 +45,39 @@ resource "aws_security_group" "db_sg" {
   }
 }
 
-# Create database for hive
-resource "aws_db_instance" "apiarydb" {
-  allocated_storage       = "${var.db_size}"
-  storage_type            = "${var.db_storage_type}"
-  engine                  = "aurora"
-  instance_class          = "${var.db_type}"
-  name                    = "apiary-metastore-db"
-  identifier_prefix       = "apiary"
-  username                = "${data.vault_generic_secret.apiarydb_master_user.data["username"]}"
-  password                = "${data.vault_generic_secret.apiarydb_master_user.data["password"]}"
-  db_subnet_group_name    = "apiarydbsg"
-  backup_retention_period = "${var.db_backup_retention}"
-  publicly_accessible     = "false"
-  multi_az                = "true"
-  vpc_security_group_ids  = ["${aws_security_group.db_sg.id}"]
-  tags                    = "${var.apiary_tags}"
+resource "aws_rds_cluster" "apiary_cluster" {
+
+    cluster_identifier            = "apiary-cluster"
+    database_name                 = "apiary"
+    master_username               = "${data.vault_generic_secret.apiarydb_master_user.data["username"]}"
+    master_password               = "${data.vault_generic_secret.apiarydb_master_user.data["password"]}"
+    backup_retention_period       = "${var.db_backup_retention}"
+    preferred_backup_window       = "02:00-03:00"
```

Comment:
What timezone are these in?

Reply:
these are in UTC, pushed a new commit to use variable for this.